### DISCUSSION TITLE
v0.8.1

### DISCUSSION THREAD

**Discussion by af** [af/hyperpaper-planner#15] [Announcements]
- First pass at support for the Supernote (space above the header for its menu)
- Move Day template's `AGENDA` label to the right to allow 6am entries
- Add faint lines to the dotted Day template's agenda section
- Various font-size and spacing tweaks

<hr /><em>This discussion was created from the release <a href='https://github.com/af/hyperpaper-planner/releases/tag/'>v0.8.1</a>.</em>

**Comment by Tunabus** [comment]
Just ordered and while I was filling out Google Form my browser closed out. How do I got about choosing my customizations?
